phalcon/mvc/model/criteriainterface.zep

```diff
@@ -192,6 +193,6 @@ interface CriteriaInterface
 	/**
 	 * Executes a find using the parameters built with the criteria
 	 */
-	public function execute() -> <ResultsetInterface>;
+	public function execute(<TransactionInterface> transaction = null) -> <ResultsetInterface>;
```

Comment:
hmm, i do understand that and I don't argue about that. But since the criteria implements the interface I need another approach

I didn't have time for a sophisticated solution basically I was implementing an API and I needed the unitests withouth rebuilding the database everytime. 

the quickest solution would be just to add a transaction property + getter and setter to criteria ? would this suffice your needs ? it would not change the method signatures and add functionality instead of changing it ?